blocks/api/test/parser.js

```diff
@@ -223,7 +223,7 @@ describe( 'block parser', () => {
 			);
 			expect( block.name ).toBe( 'core/unknown-block' );
 			expect( block.attributes.fruit ).toBe( 'Bananas' );
-			expect( block.attributes.content ).toContain( 'core/test-block' );
```

Comment:
This test is kind of weird, I don't understand why we were receiving `core/test-block` in the content attributes and why we're receiving `wp:test-block` right now. (Both seem odd if we look at the definition of the unknown block)

Reply:
This was changed in #2513.